pages/instructorAssessmentQuestionStatistics/instructorAssessmentQuestionStatistics.ejs

```diff
@@ -117,6 +117,7 @@
           <table id="questionsTable" class="table table-sm table-hover tablesorter">
             <thead>
               <tr>
+                <th class="text-center" width="50">Thumbnail</th>
                 <th class="text-center">Question</th>
```

Comment:
```suggestion
                <th colspan="2">Question</th>
```

I think we should just include the thumbnail under the "Question" label here, and left-justify the header.

I generally think we should do this on all pages where we are just listing questions, and the user doesn't really care about the specific parts of the question. This does not apply to `pages/instructorQuestions`, because there we are really separating out thumbnail/title/qid/etc. But almost all other pages should show the thumbnail as part of the "Question" column (even though it might actually be a separate column in the table HTML).